Topics/Professionalism.md

```diff
@@ -67,6 +67,29 @@ Professionalism is essential in all aspects of software engineering, and maintai
 
+### Handling criticism and feedback
+
+Before we begin, you should recognize that we deliberately chose to use the word “criticism” in the title of this section. We intend to get you to understand that the strategies we list below are not just applicable when receiving nice and polite “feedback” but also harsher “criticism.” The goal is to leave the critic (giver of feedback) with a better impression of you. This section is quite long because, arguably, the marker of someone with outstanding professionalism is that they are masters of managing criticism.
+
+---
+
+First, we should ensure our attitudes about criticism and feedback are proper. Understand that when a co-worker, client, boss, manager, senior developer, or whoever gives you criticism or feedback, they may not fully understand what they want you to do. In other words, treat criticism and feedback as an open conversation, similar to how we learned in lectures on how to build solutions for clients. When someone approaches you with criticism or feedback, they have a problem in mind for which their fix is not necessarily going to solve their problem. As a professional, it is up to you to determine the *exact* problem they have and to communicate with them to arrive at a solution that you can also be happy with (**this last part is super important!!!** Being happy with the solution will help you give off the impression that you genuinely cared about their opinion and are not just reluctantly resolving the situation). Once you’ve solved the problem and implemented the changes, go back and check to see if there are other areas of your code or your work where you can apply the same feedback.
+
+1. **Ask for suggestions or examples of how to solve the issue when unsure of what feedback means or entails.** For example, suppose your boss tells you that you are not engaged enough in online meetings (vague!). Ask your boss for suggestions on how to be more engaged. Your boss might say they want you to talk more (still vague!), but at least you now sense what your boss means. Moving forward, you might want to say “Hello” to everyone when you join the call or give signs of affirmation throughout the meeting, such as nodding your head. You can then follow up with your boss to ask if they feel you have improved. If they say no, then you try something else and ask for more clarification until you’ve narrowed down what exactly was the problem and its corresponding solution.
+2. **Avoid being overly defensive and overwhelmed by taking a step back if the criticism seems too harsh.** You can step back in various ways, but we recommend being honest and communicating that you need a few minutes to compose yourself. This pause will also give the critic time to adjust their tone and thoughts so that the conversation can be more productive and beneficial to everyone involved.
```

Comment:
Good point! I believe if this is something that you find yourself repeatedly having to do, it may indicate a bigger issue with perhaps the way the feedback is being given, or the way it's being received. Either way, there may be an escalation needed here, such as the one mentioned in point 7.

Reply:
Okay, glad to hear we're on the same page!